demisto_sdk/__main__.py

```diff
@@ -285,23 +286,25 @@ def validate(config, **kwargs):
         print_error(f'File {file_path} was not found')
         return 1
     else:
-        is_external_repo = tools.is_external_repository()
-
-        validator = ValidateManager(is_backward_check=not kwargs['no_backward_comp'],
-                                    only_committed_files=kwargs['post_commit'], prev_ver=kwargs['prev_ver'],
-                                    skip_conf_json=kwargs['no_conf_json'], use_git=kwargs['use_git'],
-                                    file_path=file_path,
-                                    validate_all=kwargs.get('validate_all'),
-                                    validate_id_set=kwargs['id_set'],
-                                    skip_pack_rn_validation=kwargs['skip_pack_release_notes'],
-                                    print_ignored_errors=kwargs['print_ignored_errors'],
-                                    is_external_repo=is_external_repo,
-                                    print_ignored_files=kwargs['print_ignored_files'],
-                                    no_docker_checks=kwargs['no_docker_checks'],
-                                    silence_init_prints=kwargs['silence_init_prints'],
-                                    skip_dependencies=kwargs['skip_pack_dependencies'],
-                                    id_set_path=kwargs.get('id_set_path'))
-        return validator.run_validation()
+        try:
+            is_external_repo = tools.is_external_repository()
+            validator = ValidateManager(is_backward_check=not kwargs['no_backward_comp'],
+                                        only_committed_files=kwargs['post_commit'], prev_ver=kwargs['prev_ver'],
+                                        skip_conf_json=kwargs['no_conf_json'], use_git=kwargs['use_git'],
+                                        file_path=file_path,
+                                        validate_all=kwargs.get('validate_all'),
+                                        validate_id_set=kwargs['id_set'],
+                                        skip_pack_rn_validation=kwargs['skip_pack_release_notes'],
+                                        print_ignored_errors=kwargs['print_ignored_errors'],
+                                        is_external_repo=is_external_repo,
+                                        print_ignored_files=kwargs['print_ignored_files'],
+                                        no_docker_checks=kwargs['no_docker_checks'],
+                                        silence_init_prints=kwargs['silence_init_prints'],
+                                        skip_dependencies=kwargs['skip_pack_dependencies'],
+                                        id_set_path=kwargs.get('id_set_path'))
+            return validator.run_validation()
+        except (git.InvalidGitRepositoryError, git.NoSuchPathError, FileNotFoundError):
+            print_error("You are running demisto-sdk validate not in the content repository.")
```

Comment:
```suggestion
            print_error("You are running not `demisto-sdk validate` in the content repository.")
```

maybe add some guidance as to how to solve this?

Reply:
ok